src/pages/sign-in/index.tsx

```diff
@@ -50,30 +84,23 @@ const Index = () => {
             Sign in to your account
           </span>
         </div>
-        <div className="w-full">
-          <SignInTextField
-            placeholder={"Email"}
-            type={"text"}
-            dataCy={"sign-in-email"}
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) => setEmail(e.target.value)}
-          />
-        </div>
-        <div className="w-full">
-          <SignInTextField
-            placeholder={"Password"}
-            type={"password"}
-            dataCy={"sign-in-password"}
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) => setPassword(e.target.value)}
-          />
-        </div>
-        {showErrorPopup && (
-          <ErrorPopup
-            closePopup={() => setErrorPopup(false)}
-            text={errorPopupText}
-            dataCy={"error-popup-sign-in"}
-          />
-        )}
-        <MainButton onClick={signIn} text={"Sign in"} dataCy={"sign-in-submit-btn"} />
+        {signInFormData.map(({ dataCy, placeholder, validationOptions, tag, type }) => {
```

Comment:
Putting this in a `<form>` will allow user to move between fields and submit the form by using their enter/return key,